src/index.js

```diff
@@ -1,77 +1,80 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from 'react';
```

Comment:
So we're using standardjs which will fail on this. I think to undo the semicolons you can just run `yarn standard --fix`. We should add a test step to run standard, so that it won't build if standard doesn't pass (this wasn't so important when it was just me contributing and it was all just flagged in my editor)

Reply:
Gotcha.  Personally, I do prefer the semicolons, but it doesn't make a huge difference to me.  Babel will inevitably add them afterwards anyhow, so I'm fine to keep them.  If this is a big deal, we can override it in the options here: https://prettier.io/docs/en/options.html#semicolons

Do note that it doesn't matter what your personal style choice or editor configuration is.  Prettier is set to run before every commit... so regardless of how you write code, it will push with Prettier's formatting (which currently includes semicolons).

Reply:
Got it, read the other PR also and saw the prettier stuff, was missing that context. I like prettier and use it on other projects but I would prefer to stick with standardjs here just because that's what the project started with and I'd prefer to avoid the massive reformatting commit that mangles the diffs a bit when tracing back how things changed functionally over time. Make sense?